### DISCUSSION TITLE
filed type str with input int

### DISCUSSION THREAD

**Discussion by zlpmetyou** [pydantic/pydantic#10232] [Question] [answered]
i think all types  of python can convert to str, but when str field input int, i got an error :string_type. Can pydantic handle it automatically

**Comment by Viicos** [comment, marked as answer]
You are looking for https://docs.pydantic.dev/latest/api/config/#pydantic.config.ConfigDict.coerce_numbers_to_str.

**Reply by zlpmetyou** [reply]
thank you. that's what I want
